src/main/java/com/airiline/tickets/service/impl/TicketServiceImpl.java

```diff
@@ -50,7 +50,7 @@ public void deleteById(Long id) {
         ticketRepository.delete(findById(id));
     }
 
-    private Ticket findById(Long id) {
+    public Ticket findById(Long id) {
```

Comment:
I think we can keep it private

Reply:
Yes, you are right, I made it public in order to be able to simulate ticket creation. Than I just added a to do comment  for this and that's why it remained public. I will change it back.